GEOSldas_App/ldas_setup

```diff
@@ -152,40 +161,77 @@ class LDASsetup:
         if cmdLineArgs['nymdb'] != 'None' and cmdLineArgs['nhmsb'] != 'None' :
             self.date  = f"{self.nymdb} {self.nhmsb}"
```

Comment:
self.date is not used and is not necessarr to be define. Combine the two line to one